### PR TITLE
SSCS-10243 make dwpIsOfficerAttending attribute visible

### PR DESCRIPTION
**Pull request by augustio**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-10243

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by nickwest1 — APPROVED** [review]

File: benefit/data/sheets/WorkBasketInputFields/WorkBasketInputFields.json [hunk 1]
```diff
@@ -59,5 +60,6 @@
...
   {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "CaseFieldID": "reasonableAdjustmentsOutstanding", "Label": "Alternative letter format required", "DisplayOrder": 15},
   {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "CaseFieldID": "hasUnprocessedAudioVideoEvidence", "Label": "Does case have unprocessed audio/video evidence?", "DisplayOrder": 16},
-  {"LiveFrom": "16/12/2021", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "CaseFieldID": "dwpDueDate", "Label": "FTA response due date", "DisplayOrder": 17}
-]
+  {"LiveFrom": "16/12/2021", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "CaseFieldID": "dwpDueDate", "Label": "FTA response due date", "DisplayOrder": 17},
+  {"LiveFrom": "10/10/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "CaseFieldID": "dwpIsOfficerAttending", "Label": "Is the Presenting officer attending", "DisplayOrder": 18}
```
Comment (MohammedHMCTS) [inline benefit/data/sheets/WorkBasketInputFields/WorkBasketInputFields.json:64]: If this is new we should add the current date in the LiveFrom such as 25/05/2022

File: benefit/data/sheets/WorkBasketResultFields/WorkBasketResultFields.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
   {"LiveFrom": "11/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-dwpresponsewriter", "CaseFieldID": "dwpState", "Label": "State", "DisplayOrder": 6},
   {"LiveFrom": "11/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-dwpresponsewriter", "CaseFieldID": "caseCreated", "Label": "Case created date", "DisplayOrder": 7},
   {"LiveFrom": "11/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-dwpresponsewriter", "CaseFieldID": "dateSentToDwp", "Label": "FTA referral date", "DisplayOrder": 8, "ResultsOrdering": "1:ASC"},
+  {"LiveFrom": "10/10/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-dwpresponsewriter", "CaseFieldID": "dwpIsOfficerAttending", "Label": "Is the Presenting officer attending", "DisplayOrder": 9},
```
Comment (MohammedHMCTS) [inline benefit/data/sheets/WorkBasketResultFields/WorkBasketResultFields.json:18]: This date should also be updated

File: benefit/data/sheets/WorkBasketResultFields/WorkBasketResultFields.json [hunk 3]
```diff
@@ -37,5 +38,6 @@
...
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "CaseFieldID": "interlocReviewState", "Label": "Interlocutory review state", "DisplayOrder": 8},
   {"LiveFrom": "25/09/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "CaseFieldID": "linkedCasesBoolean", "Label": "Has related appeal(s)?", "DisplayOrder": 9},
-  {"LiveFrom": "25/09/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "CaseFieldID": "[LAST_STATE_MODIFIED_DATE]", "Label": "Date of state change", "DisplayOrder": 10, "ResultsOrdering": "1:ASC"}
+  {"LiveFrom": "25/09/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "CaseFieldID": "[LAST_STATE_MODIFIED_DATE]", "Label": "Date of state change", "DisplayOrder": 10, "ResultsOrdering": "1:ASC"},
+  {"LiveFrom": "10/10/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "CaseFieldID": "dwpIsOfficerAttending", "Label": "Is the Presenting officer attending", "DisplayOrder": 11}
```
Comment (MohammedHMCTS) [inline benefit/data/sheets/WorkBasketResultFields/WorkBasketResultFields.json:42]: Same here with the date

**Review by MohammedHMCTS — CHANGES_REQUESTED** [review]

File: benefit/data/sheets/WorkBasketInputFields/WorkBasketInputFields.json (hunk 1 above)
Comment (johnoneill46) [reply benefit/data/sheets/WorkBasketInputFields/WorkBasketInputFields.json:64]: Shouldn't matter though right? LiveFrom dates are deprecated by CCD anyway and aren't actually used for anything?

Comment (MohammedHMCTS) [reply benefit/data/sheets/WorkBasketInputFields/WorkBasketInputFields.json:64]: It doesn't actually effect anything but gives good insight on when we added stuff to trace it back if needed.

**Comment by MohammedHMCTS** [comment]
Please upgrade Version for tracking before merging or can cause issues.

**Comment by MohammedHMCTS** [comment]
Please upgrade Version for tracking before merging or can cause issues.

**Review by nickwest1 — APPROVED** [review]

**Review by osheaf01 — APPROVED** [review]
LGTM

**Pull request merged**